src/main/java/org/openstreetmap/atlas/generator/dataframe/README.md

```diff
@@ -0,0 +1,29 @@
+# `AtlasDataFrame`
+## Overview
+`AtlasDataFrame` is a class that houses methods that convert an atlas RDD to a `DataFrame`. This allows for easily querying the data of an atlas from within an active spark session. Each entity is separated into its own dataframe, and supports its own schema.
+
+### Schemas
+
+Areas:
+
+`identifier` `geometry` `bounds` `tags` `relations`
```

Comment:
Here I would make that a markdown table, with the type below, like:
`identifier` | `geometry` | `bounds` | `tags` | `relations`
--|--|--|--|--
String|String|String|Map|Array

And the same for the ones below.